lib/active_record/connection_adapters/oracle_enhanced_adapter.rb

```diff
@@ -73,6 +73,12 @@ def oracle_enhanced_connection(config) #:nodoc:
         ConnectionAdapters::OracleEnhancedAdapter.new(
           ConnectionAdapters::OracleEnhanced::Connection.create(config), logger, config)
       end
+    rescue OCIError => error
```

Comment:
Oops. It should also support JDBC.

> expected ActiveRecord::NoDatabaseError, got #<NameError: uninitialized constant ActiveRecord::ConnectionHandling::OCIError> with backtrace:

https://travis-ci.org/rsim/oracle-enhanced/jobs/556237777#L651